leits/MeetingBar
# feat(events): allow to hide tentative events #515

### Status
**READY**

### Description
- allows to show, hide or mark tentative events as inactive. Tentative events are maybe events where the recipient do not know if he will attend.


## Checklist

### Steps to Test or Reproduce
- Send an event from a second calendar provider 
- Answer event with "tentative/maybe"
- Check the menubar /statusmenu by configure the new setting
